lib/btrfs.coffee

```diff
@@ -1,13 +1,11 @@
-{ spawn } = require 'child_process'
-
-utils = require './utils'
+{ spawn } = require './utils'
 
 # Returns a promise that settles after the subvolume `subvol` has been deleted
 exports.deleteSubvolAsync = (subvol) ->
-	utils.waitPidAsync(spawn('btrfs', [ 'subvolume', 'delete', subvol]))
+	spawn('btrfs', [ 'subvolume', 'delete', subvol]).waitAsync()
```

Comment:
Can you make the spacing within the `[]` consistent - there's a space after the opening [ but not before the ]